pegasus/sites.v3/code.org/views/maker_curriculum_circuit_playground.haml

```diff
@@ -0,0 +1,17 @@
+.action-block__wrapper.action-block__wrapper--two-col
+  .action-block.action-block--one-col
+    %p.overline Grades 6-12
+    %h3 Creating Apps with Devices
+    %figure.video-responsive
+      %div
+        %iframe{src: "https://www.youtube-nocookie.com/embed/9_Z1hvx22us", title: "Creating Apps with Devices", frameborder: "0", allowfullscreen: "true"}
+    %p Explore the role of physical devices in computing. Using App Lab and Adafruit's Circuit Playground, students develop programs that utilize the same hardware inputs and outputs that you see in the smart devices.
+    %p{style: "height: 50px"} <strong>Duration:</strong> 25 hours
+    %a.link-button{href: explore_unit_link, aria:{label: "Explore the Creating Apps with Devices unit"}} Explore the unit
```

Comment:
TIL about this syntax for aria label! Looks so bizarre. I'm used to `'aria-label'=>` or `'aria-label':` ... haml ...

Reply:
But also love this use of the label! 🥳

Reply:
Oh that makes a lot of sense! I see it in other places in the code base, so I'm good with it –– we don't have norms yet for haml, so I don't think it makes sense to try to "fix" things without clear direction as to what our norms are.

Reply:
Ok cool, sounds good! 

Side note: I like this version bc the color code formatting making it's easier to see it's an attribute, so maybe that's something to consider for when we norm:
<img width="959" alt="Screenshot 2023-03-22 at 10 49 38 AM" src="https://user-images.githubusercontent.com/9256643/226993602-3d30b291-2eab-4de1-8a2b-df217f77a9c7.png">